### ISSUE TITLE
setSSLVerification type hint breaks CA cert validation

### ISSUE THREAD

**Issue by slepp** [elastic/elasticsearch-php#916]
### Summary of problem or feature request

In 6.7.1, the option to set the CA certificate for request TLS verification is allowed to be `false` or a string path to a CA certificate.

In 7.0, this has been changed to a type hint of `bool` on the method, which casts the string to `true` and self-signed certificates are no longer used.

### Code snippet of problem

```php
$builder->setSSLVerification('/etc/ca/ca.pem'); // should be a string, converts to true
```

### System details

- Linux
- PHP 7.2
- ES-PHP 7.0.2
- ES 7.2.0

**Comment by ezimuel** [comment]
@slepp I provided the PR #917 to fix this. I'll release it soon in 7.1.1

**Comment by ezimuel** [comment]
@slepp just released [elasticsearch 7.1.1](https://github.com/elastic/elasticsearch-php/releases/tag/v7.1.1) that fixes the issue. Thanks for reporting this issue!
